Xamarin.Forms.Platform.GTK/Renderers/ImageRenderer.cs

```diff
@@ -64,6 +67,14 @@ protected override void OnElementPropertyChanged(object sender, PropertyChangedE
 				SetOpacity();
 			else if (e.PropertyName == Image.AspectProperty.PropertyName)
 				SetAspect();
+			else if (e.PropertyName == Image.ScaleProperty.PropertyName)
+				SetScale();
```

Comment:
Thanks for the contribution. Mnn, could we move this implementation to `VisualElementTracker` https://github.com/xamarin/Xamarin.Forms/blob/master/Xamarin.Forms.Platform.GTK/VisualElementTracker.cs#L279?. In this way, the transformation can be applied to any visual element.

Reply:
The implementation is specifically designed for rotation and scaling of images and uses Cairo etc. to do so. I don't think it's easily portable to support any visual element.

Reply:
@jsuarezruiz ping? I think Bobface raises a good point here. BTW, issue #5322 has been fixed in master branch (4.7.x?), which was related to this.

Reply:
Ideally this should be managed by `VisualElementTracker` for all the Views. I understand that the implementation is not easy for everything, if we allow this it would be an exception (manage transformations of a specific view like an Image in the view renderer). Otherwise, it works correctly.